src/ralph/virtual/models.py

```diff
@@ -35,11 +35,26 @@
 logger = logging.getLogger(__name__)
 
 
+class CloudSyncProcessor(AdminAbsoluteUrlMixin, models.Model):
```

Comment:
what's the point of this model? Couldn't we point processor by name in the url?

Reply:
I wanter users to have a convenient admin pages where they can select processors from the list of available ones. On the other hand binding the field directly to the list of available entry points would require an automatic synchronisation of the data in the DB with the list of entry points.

It's a good point to extend the router with a check whether or not the specified processor is actually available.

Reply:
But if processor is pointed in the url, user will "select" it in hermes console (by entering url). Right now you still have to sync data between db and code (ex. what if I change the name of the processor or remove one)? The compromise here would be storying `sync_event_processor` as string in `CloudProvider` model :)